SpaceDash/Controller/ViewControllers/HomeViewController.swift

```diff
@@ -23,6 +23,8 @@ class HomeViewController: UIViewController,NetworkManagerDelegate {
     var constants : Constants.HomeView?
     var senderView : String = ""
     
+    let tenativeDetail = "This is the tentative launch date and subjected to change"
```

Comment:
Add the strings to the constants file under the respective struct and use the resulting constants. Also, add tests under the constants check Unit test.

Reply:
sure will do